Menus/MainMenu/MainMenu.cs

```diff
@@ -97,7 +99,7 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
 			consumeControl(mainMenuInput.blockY);
 
 			const float kFlickDeltaThreshold = 0.5f;
-			if ((proxyType != typeof(ViveProxy) && Mathf.Abs(rotationInput) >= kFlickDeltaThreshold && Mathf.Abs(m_LastRotationInput) < kFlickDeltaThreshold)
+			if (proxyType != typeof(ViveProxy) && Mathf.Abs(rotationInput) >= kFlickDeltaThreshold && Mathf.Abs(m_LastRotationInput) < kFlickDeltaThreshold
```

Comment:
No change to the logic. R# was telling me those parenthesis were redundant. There's an || on the next line, but everything in this line are &&s